.reuse/dep5

```diff
@@ -0,0 +1,8 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: reference-design-air-quality
+Upstream-Contact: Golioth, Inc. <https://golioth.io>
+Source: https://github.com/golioth/reference-design-air-quality
+
+Files: *.jpg *.jpeg *.png
+Copyright: Golioth, Inc. <https://golioth.io>
+License: Apache-2.0
```

Comment:
@ChrisGammell everything in this repo is licensed Apache-2.0, so I added the images under the same license. However, we can use a different license for these images if you want.